src/main/java/server/directory/DirectoryIndex.java

```diff
@@ -35,7 +35,7 @@ private String currentFolder() {
     private String contentsList() {
         StringBuilder contentsList = new StringBuilder("<ul>");
 
-        for(File file : sortedContents()) {
+        for (File file : sortedContents()) {
             String href = file.getPath().replace(DefaultDirectory.PATH, "");
```

Comment:
I must have missed this in a previous review, but do you think it's possible to remove the global mutable state from `DefaultDirectory` (`PATH`)?